src/slurmjobs.rs

```diff
@@ -5,36 +5,35 @@ use crate::output;
 use crate::systemapi;
 use crate::time;
 
-use lazy_static::lazy_static;
+use once_cell::sync::Lazy;
 
 #[cfg(test)]
 use std::cmp::min;
 use std::collections::HashSet;
 use std::io;
 
-lazy_static! {
-    // The job states we are interested in collecting information about, notably PENDING and RUNNING
-    // are not here by default but will be added if the --deluge option is given.
+// The job states we are interested in collecting information about, notably PENDING and RUNNING
+// are not here by default but will be added if the --deluge option is given.
 
-    //+ignore-strings
```

Comment:
It would have been necessary to keep the ignore-strings comments previously (they are for a test case), but with the latest patch we landed it's fine to remove them.

Reply:
Ah! Good to know. Thanks.